include/posix/sys/time.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2019 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_POSIX_SYS_TIME_H_
+#define ZEPHYR_INCLUDE_POSIX_SYS_TIME_H_
+
+#ifdef CONFIG_NEWLIB_LIBC
+/* Kludge to support outdated newlib version as used in SDK 0.10 for Xtensa */
+#include <newlib.h>
+
+#ifdef __NEWLIB__
```

Comment:
Can you add some more comments here about what's going on.  I'm guessing this is all because of xtensa's version of newlib being very old.

Reply:
Comment 2 lines above of "Kludge to support outdated newlib version as used in SDK 0.10 for Xtensa" not enough?